From 47bfee83f8f015ed342f8606bff5b262b8c23925 Mon Sep 17 00:00:00 2001
From: Antonio Ospite <ospite@studenti.unina.it>
Date: Fri, 6 Jan 2012 13:07:22 +0100
Subject: [PATCH 1/3] debian/patches: refresh 0008-update_manpages.patch
X-Face: z*RaLf`X<@C75u6Ig9}{oW$H;1_\2t5)({*|jhM<pyWR#k60!#=#>/Vb;]yA5<GWI5`6u&+
 ;6b'@y|8w"wB;4/e!7wYYrcqdJFY,~%Gk_4]cq$Ei/7<j&N3ah(m`ku?pX.&+~:_/wC~dwn^)MizBG
 !pE^+iDQQ1yC6^,)YDKkxDd!T>\I~93>J<_`<4)A{':UrE

Now git buildpackage works again.
---
 debian/patches/0008-update_manpages.patch |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/patches/0008-update_manpages.patch b/debian/patches/0008-update_manpages.patch
index 9099e2e..8258093 100644
--- a/debian/patches/0008-update_manpages.patch
+++ b/debian/patches/0008-update_manpages.patch
@@ -159,8 +159,8 @@ index ec77aef..7651027 100644
  			DESTINATION ${TARGETDIR}
  		)
  
-@@ -361,7 +362,9 @@ if(UNIX AND NOT APPLE)
-             DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/blender/scripts
+@@ -361,7 +362,9 @@
+ 			DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/blender/scripts
  		)
  		install(
 -			FILES ${CMAKE_SOURCE_DIR}/doc/manpage/blender.1
-- 
1.7.8.2


From 282d3e009ef2f54d8d4f1eaa5bc01897cd85dcac Mon Sep 17 00:00:00 2001
From: Antonio Ospite <ospite@studenti.unina.it>
Date: Fri, 6 Jan 2012 14:31:50 +0100
Subject: [PATCH 2/3] debian/patches: don't use version number in the
 system_path
X-Face: z*RaLf`X<@C75u6Ig9}{oW$H;1_\2t5)({*|jhM<pyWR#k60!#=#>/Vb;]yA5<GWI5`6u&+
 ;6b'@y|8w"wB;4/e!7wYYrcqdJFY,~%Gk_4]cq$Ei/7<j&N3ah(m`ku?pX.&+~:_/wC~dwn^)MizBG
 !pE^+iDQQ1yC6^,)YDKkxDd!T>\I~93>J<_`<4)A{':UrE

Add 0010-Don-t-use-version-number-in-the-system_path.patch

Global installation expects system path to be something like
/usr/lib/blender/scripts/ without version numbers.

Thanks: Luka Frelih <luka@ljudmila.org>
Closes: #654395
---
 ...n-t-use-version-number-in-the-system_path.patch |   39 ++++++++++++++++++++
 debian/patches/series                              |    1 +
 2 files changed, 40 insertions(+), 0 deletions(-)
 create mode 100644 debian/patches/0010-Don-t-use-version-number-in-the-system_path.patch

diff --git a/debian/patches/0010-Don-t-use-version-number-in-the-system_path.patch b/debian/patches/0010-Don-t-use-version-number-in-the-system_path.patch
new file mode 100644
index 0000000..2092305
--- /dev/null
+++ b/debian/patches/0010-Don-t-use-version-number-in-the-system_path.patch
@@ -0,0 +1,39 @@
+From 93a84599a36fb10588d4c93bc4c465d179b5ee6e Mon Sep 17 00:00:00 2001
+From: Antonio Ospite <ospite@studenti.unina.it>
+Date: Fri, 6 Jan 2012 14:31:50 +0100
+Subject: [PATCH] Don't use version number in the system_path.
+X-Face: z*RaLf`X<@C75u6Ig9}{oW$H;1_\2t5)({*|jhM<pyWR#k60!#=#>/Vb;]yA5<GWI5`6u&+
+ ;6b'@y|8w"wB;4/e!7wYYrcqdJFY,~%Gk_4]cq$Ei/7<j&N3ah(m`ku?pX.&+~:_/wC~dwn^)MizBG
+ !pE^+iDQQ1yC6^,)YDKkxDd!T>\I~93>J<_`<4)A{':UrE
+
+Global installation expects system patch to be something like
+/usr/lib/blender/scripts/ without version numbers.
+---
+ source/blender/blenlib/intern/path_util.c |    4 ++--
+ 1 files changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/source/blender/blenlib/intern/path_util.c b/source/blender/blenlib/intern/path_util.c
+index e79d850..bc6f6e6 100644
+--- a/source/blender/blenlib/intern/path_util.c
++++ b/source/blender/blenlib/intern/path_util.c
+@@ -89,7 +89,7 @@
+ #  else /* new XDG ~/blender/.config/ */
+ #    define BLENDER_USER_FORMAT			"%s/blender/%s"
+ #  endif // WITH_XDG_USER_DIRS
+-#  define BLENDER_SYSTEM_FORMAT			"%s/blender/%s"
++#  define BLENDER_SYSTEM_FORMAT			"%s/blender"
+ #endif
+ 
+ /* local */
+@@ -1025,7 +1025,7 @@ static int get_path_system(char *targetpath, const char *folder_name, const char
+ 
+ 	system_base_path = (const char *)GHOST_getSystemDir();
+ 	if (system_base_path) {
+-		BLI_snprintf(system_path, FILE_MAX, BLENDER_SYSTEM_FORMAT, system_base_path, blender_version_decimal(ver));
++		BLI_snprintf(system_path, FILE_MAX, BLENDER_SYSTEM_FORMAT, system_base_path);
+ 	}
+ 	
+ 	if(!system_path[0])
+-- 
+1.7.8.2
+
diff --git a/debian/patches/series b/debian/patches/series
index 41b4c6a..68acb7b 100644
--- a/debian/patches/series
+++ b/debian/patches/series
@@ -7,3 +7,4 @@
 0007-locales_directory_install.patch
 0008-update_manpages.patch
 0009-fix_FTBFS_with_ffmpeg_debian.patch
+0010-Don-t-use-version-number-in-the-system_path.patch
-- 
1.7.8.2

